add existing dir warning and how to fix it (#187)
oceanhackweek/oceanhackweek.github.io: resources/prep/jupyterhub.md

```{warning}

> If you start by using the nbgitpuller link, and then switch to using git directly, or if you already have a copy of the repository in our account, using the nbgitpuller link again will most likely lead to [unpredictable results](https://jupyterhub.github.io/nbgitpuller/#when-to-use-nbgitpuller).

Reviewer: A fix ("our" > "your"), tweaks to highlight nbgitpuller, and a clarification about the copy of the repository:
```suggestion
If you start by using the `nbgitpuller` link and then switch to using git directly, or if you already have a copy of the repository in your account from a previous OHW, using the `nbgitpuller` link again will most likely lead to [unpredictable results](https://jupyterhub.github.io/nbgitpuller/#when-to-use-nbgitpuller).
```

> If you start by using the nbgitpuller link, and then switch to using git directly, or if you already have a copy of the repository in our account, using the nbgitpuller link again will most likely lead to [unpredictable results](https://jupyterhub.github.io/nbgitpuller/#when-to-use-nbgitpuller).
> 
> This can be fixed by [removing the `ohw-tutorials` directory](https://imgs.xkcd.com/comics/git.png) and using nbgitpuller again.

Reviewer 2: ```suggestion
If you start by using the nbgitpuller link, and then switch to using git directly, or if you already have a copy of the repository in our account, using the nbgitpuller link again will most likely lead to [unpredictable results](https://jupyterhub.github.io/nbgitpuller/#when-to-use-nbgitpuller).

This can be fixed by [removing, renaming, or moving the `ohw-tutorials` directory](https://imgs.xkcd.com/comics/git.png) and using nbgitpuller again.
```
